client/src/components/HtmlEditorField/HtmlEditorField.js

```diff
@@ -0,0 +1,46 @@
+/* global window */
+import React from 'react';
+import Script from 'react-load-script';
+import { Component as TextField } from 'components/TextField/TextField';
+import fieldHolder from 'components/FieldHolder/FieldHolder';
+
+class HtmlEditorField extends TextField {
+  constructor(props) {
+    super(props);
+    this.state = {
+        isReady: false
+    };
+    this.handleReady = this.handleReady.bind(this);
+  }
+
+  getInputProps() {
+    return {
+      ...super.getInputProps(),
+      ...this.props.data.config,
```

Comment:
what kind of config props are spread here?
this adds a bit of ambiguity of what is expected to happen, could `HtmlEditorField` contain these props instead of getting them from the `data.config` prop?

Reply:
`data-config`, `data-editor` - both of which are required for TinyMCE to bootstrap.

Reply:
as above, data.config is now data.attributes - along with added docblocks, I hope confusing is reduced to any new reader :)